plugins/snippets/luasnip/default.nix

```diff
@@ -133,6 +133,27 @@ in
       '';
       type = types.listOf loaderSubmodule;
     };
+
+    filetypeExtend = mkOption {
+      default = { };
+      type = with types; attrsOf (listOf str);
```

Comment:
I expect this is a file-wide issue, but this doesn't support rawLua.

_Ideally_ the whole list or any list item could be rawLua. We can save this for a future work though if the issue applies to other options in this module.